cumulus/etc/iproute2/rt_protos.d/frr.conf

```diff
@@ -0,0 +1,8 @@
+# Additional protocol strings defined by quagga for each of its daemons
```

Comment:
This is _massively_ user-visible and likely to break some user setups that look at the protocol value -- not appropriate for stable/2.0 IMHO.  Even on master, I'd make it (startup/cmdline) configurable...